headers/Response.hpp

```diff
@@ -0,0 +1,56 @@
+/* ************************************************************************** */
+/*                                                                            */
+/*                                                        :::      ::::::::   */
+/*   Response.hpp                                       :+:      :+:    :+:   */
+/*                                                    +:+ +:+         +:+     */
+/*   By: fkoolhov <fkoolhov@student.42.fr>          +#+  +:+       +#+        */
+/*                                                +#+#+#+#+#+   +#+           */
+/*   Created: 2024/04/23 17:04:25 by fkoolhov          #+#    #+#             */
+/*   Updated: 2024/04/25 14:47:54 by fkoolhov         ###   ########.fr       */
+/*                                                                            */
+/* ************************************************************************** */
+
+#ifndef RESPONSE_HPP
+# define RESPONSE_HPP
+
+# include "webserv.hpp"
+# include "Request.hpp"
+
+class Response
+{
+	private:
+		std::unique_ptr<Request>				_request;
+		short									_statusCode;
+		std::string								_reasonPhrase;
+		std::map<std::string, std::string>		_responseHeaders;
+		std::string								_header; // only the response header (http version, status code, reason phrase, response headers)
```

Comment:
in the request class I save headers as a std::map<std::string, std::string>. Maybe also useful here?